alert/teams.go

```diff
@@ -186,12 +188,35 @@ func CreateCards(promAlert notify.WebhookMessage, webhook *PrometheusWebhook) (s
 }
 
 // SendCard sends the Teams message card
-func SendCard(webhook string, card string) (*http.Response, error) {
-	res, err := http.Post(webhook, "application/json", strings.NewReader(card))
+func SendCard(webhook string, card string, maxIdleConns int, idleConnTimeout time.Duration, tlsHandshakeTimeout time.Duration) (*http.Response, error) {
+
+	c := &http.Client{
+		Transport: &http.Transport{
+			Proxy: http.ProxyFromEnvironment,
+			DialContext: (&net.Dialer{
+				Timeout:   30 * time.Second,
```

Comment:
should we parameterize these as well and provide default values? Just a suggestion.

Reply:
I thought about doing this, but decided that on balance the three I've parametrized will be enough on the first pass and if there's a cry for more timeout tuning then we can open up an issue. Is that OK?

Reply:
Sure that's fine. I'll test your changes tomorrow and will merge it then. Thanks you!